### ISSUE TITLE
[Drilldown Menu] feature suggestion scrollTop

### ISSUE THREAD

**Issue by designerno1** [zurb/foundation-sites#8617]
for drilldown menu with very long submenu's it would be nice to have a function to jump after show,hide,closed to the top of the drilldown menu
i wrote an quick js to do this.
should we take this in the drilldown plugin, or is this a thing to do with the callbacks
```js
$('[data-drilldown]').on('open.zf.drilldown hide.zf.drilldown closed.zf.drilldown',function(){
  var animationDuration = 500, animationEasing = 'linear', scrollPos = $(this).offset().top;
  $('html, body').stop(true).animate({ scrollTop: scrollPos }, animationDuration, animationEasing);
});
```

**Comment by simonrcodrington** [comment]
Hey there @designerno1. I would like that would be something you would handle in your own callbacks (as it might be unwanted for people who don't need that functionality). Be easier to visualize if you made this into a CodePen example so everyone could see.

**Comment by designerno1** [comment]
just created a pen
http://codepen.io/anon/pen/jqxZxO
if you got a very long level 2 menu and there is a very short level 3 menu it is nice if it scrolls to the top of each drilldown element

**Comment by brettsmason** [comment]
@designerno1 I checked the CodePen and that's a really nice user experience enhancement in my opinion. I guess the best way to implement it would be via an data-scroll-top option.

I think this requires some other opinions though.

**Comment by designerno1** [comment]
ok please zurb let me know if i should code that feature into data-options feature

**Comment by Silvester20** [comment]
I like this one too +1

**Comment by kball** [comment]
Having this as a new data-option behavior sounds like a great addition for v6.3

**Comment by simonrcodrington** [comment]
I think having this an option is probably the best course of action, I like the way it works. 

This should make it's way eventually to one of the zurb guys and they will be able to let us know if this is something they are interested in.

**Comment by designerno1** [comment]
just added a PR #8633

**Comment by Owlbertz** [comment]
Can be closed since the PR has been merged, will be fixed in 6.3 then.
